examples/multimodal_v1/utils/model.py

```
@@ -0,0 +1,104 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Any, Dict, Tuple
+
+import torch
+from transformers import AutoConfig
+from utils.protocol import EncodeResponse
+from vllm import AsyncEngineArgs
+from vllm.utils import get_distributed_init_method, get_ip, get_open_port
+from transformers import AutoImageProcessor, LlavaForConditionalGeneration
```

Ruff (0.12.2)

`transformers.AutoImageProcessor` imported but unused

Remove unused import

(F401)